pom.xml

```diff
@@ -120,6 +120,11 @@
             <version>5.8.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
```

Comment:
Wrm niet `quarkus-jdbc-postgresql`? Dit is optimized voor Quarkus

Reply:
Ja ik zit het idd heb toegepast.